index.yaml

```diff
@@ -12,7 +12,7 @@ entries:
       catalog.cattle.io/release-name: fleet
     apiVersion: v2
     appVersion: 0.3.6-rc7
-    created: "2021-07-29T09:17:48.098424709-07:00"
+    created: "2021-07-30T08:54:02.599902241-07:00"
```

Comment:
Weird that all of these changed

Reply:
they always all change when 'make charts' is run